src/client/js/components/toast-alert.js

```diff
@@ -109,6 +117,11 @@ const html = `
 </output>
 `
 
+const ToastTypes = {
+  Error: 'error',
+  Success: 'success'
+}
```

Comment:
Adding additional types to the `toast-alerts`. Could potentially have `warning` or others in the future.